src/SAWScript/Crucible/LLVM/MethodSpecIR.hs

```diff
@@ -138,23 +208,49 @@ isMut = allocSpecMut . mutIso
 --------------------------------------------------------------------------------
 -- *** LLVMModule
 
+-- | An 'LLVMModule' contains an LLVM module that has been parsed from
+-- a bitcode file and translated to Crucible.
```

Comment:
It would be nice to add the paragraph about the invariant requirement for LLVMModule here to protect against future work that might be unaware of and therefore break that invariant.

Reply:
Comments added in revision 3abe2efd3dd076001799143890d64fca4bef5d67.

Reply:
Nice, thanks!